2022-02-14 16:33:16 [INFO] 获得列名:attach====成功 [case.py:14]
2022-02-14 16:33:16 [INFO] 获取入口成功，服务器：debug；数据库：debug [conf.py:10]
2022-02-14 16:33:16 [INFO] 获取服务器信息成功:http://192.168.222.154:80 [conf.py:20]
2022-02-14 16:33:16 [INFO] 读取excel文件:../excelcases/attach.xlsx====成功 [case.py:34]
2022-02-14 16:33:16 [INFO] 发送请求====成功 [case.py:83]
2022-02-14 16:33:16 [INFO] 获得响应结果中的对应参数值====成功 [case.py:50]
2022-02-14 16:33:16 [INFO] 获得响应结果中的对应参数值====成功 [case.py:50]
2022-02-14 16:33:16 [INFO] 存入公共字典,字典：{'name1': 'admin', 'passwd': '123456'}====成功 [case.py:63]
2022-02-14 16:33:16 [INFO] 发送请求====成功 [case.py:83]
2022-02-14 16:33:16 [INFO] login_01:测试成功登录====响应结果比对====成功! [case.py:100]
2022-02-14 16:33:16 [INFO] 发送请求====成功 [case.py:83]
2022-02-14 16:33:16 [INFO] 获得响应结果中的对应参数值====成功 [case.py:50]
2022-02-14 16:33:16 [INFO] 存入公共字典,字典：{'name1': 'admin', 'passwd': '123456', 'name2': 'dd'}====成功 [case.py:63]
2022-02-14 16:33:16 [INFO] 发送请求====成功 [case.py:83]
2022-02-14 16:33:16 [INFO] login_02:测试获取用户信息====响应结果比对====成功! [case.py:100]
